README.md

```diff
@@ -167,6 +167,12 @@ If you're worried that Prettier will change the correctness of your code, add `-
 This will cause Prettier to print an error message if it detects that code correctness might have changed.
 Note that `--write` cannot be used with `--debug-check`.
 
+Another useful flag is `--list-different` (or `-l`) which prints the filenames of files that are different from Prettier formatting. If there are differences the script errors out, which is useful in a CI scenario.
+
+```bash
+prettier -l './src/**/*.js'
```

Comment:
I would find this clearer if `--list-different` was used instead of `-l`.

Single quotes around the glob doesn't work on Windows, so double quotes would be better.

> ```bash
> prettier --single-quote --list-different "src/**/*.js"
> ```